package-lock.json

```diff
@@ -5275,9 +5275,9 @@
       }
     },
     "node_modules/electron-to-chromium": {
```

Comment:
@Valexr why *electron-to-chromium* here? That's not a desktop app.

Reply:
node_modules/browserslist - internal deps 🤷🏻‍♂️ Not devdeps